metadata-ingestion/src/datahub/ingestion/source/sql/mysql.py

```diff
@@ -83,3 +84,15 @@ def get_platform(self):
     def create(cls, config_dict, ctx):
         config = MySQLConfig.parse_obj(config_dict)
         return cls(config, ctx)
+
+    def add_profile_metadata(self, inspector: Inspector) -> None:
+        with inspector.engine.connect() as conn:
+            for row in conn.execute(
+                "SELECT table_schema, table_name, data_length from information_schema.tables"
+            ):
+                table_schema = row.table_schema
+                table_name = row.table_name
+                data_length = row.data_length
+                self.profile_metadata_info.dataset_name_to_storage_bytes[
+                    f"{table_schema}.{table_name}"
+                ] = data_length
```

Comment:
```suggestion
                self.profile_metadata_info.dataset_name_to_storage_bytes[
                    f"{row.table_schema}.{row.table_name}"
                ] = row.data_length
```
Slightly simpler imo